src/cpp/server/backend_metric_recorder.cc

```diff
@@ -41,16 +42,23 @@ grpc_core::TraceFlag grpc_backend_metric_trace(false, "backend_metric");
 namespace grpc {
 namespace experimental {
 
+std::unique_ptr<ServerMetricRecorder> ServerMetricRecorder::Create() {
+  return std::unique_ptr<ServerMetricRecorder>(new ServerMetricRecorder());
+}
+
 ServerMetricRecorder::ServerMetricRecorder() {
   // Starts with an empty result.
-  metric_state_ = std::make_shared<const BackendMetricDataState>();
+  metric_state_ =
```

Comment:
Nit: This can be set in the initializer list rather than in the body.

Reply:
Done.